### PR TITLE
Pull Request Ready

### PR DESCRIPTION
**Pull request by mmendez100**
Hi David, a pull request, to add me to the site, is henceforth created! Long Live Git! :)

### CODE REVIEW

File: css/mwm.css [hunk 1]
```diff
@@ -0,0 +1,139 @@
+
```
Comment (nw0428) [inline css/mwm.css]: I would prefer this file be in a folder called manuel or manuel-mendez under css

**Comment by nw0428** [comment]
One comment then LGTM

**Comment by nw0428** [comment]
@mmendez100 Could you also @ask some of your classmates to review this?

**Comment by mmendez100** [comment]
done!

**Pull request merged**